src/state_machine.zig

```diff
@@ -2327,11 +2236,6 @@ test "create/lookup/rollback transfers" {
     try test_account_balances(state_machine, 1, 100, 200, 0, 0);
     try test_account_balances(state_machine, 3, 0, 0, 110, 210);
     try expect(state_machine.get_transfer(1) == null);
-
-    for (accounts) |account| {
-        state_machine.create_account_rollback(&account);
-        try expect(state_machine.get_account(account.id) == null);
-    }
```

Comment:
We should still test this?

Reply:
Per https://github.com/tigerbeetledb/tigerbeetle/pull/192#discussion_r991473778, to simplify test setup I didn't commit the accounts, just applied them to the groove directly, so it doesn't make sense to roll back.

Reply:
Do we still have rollback coverage for accounts/transfers?

Reply:
Yes, we test account rollback in the account tests and transfer rollback in the transfer tests. This just removes account rollback from the transfer tests.